lua/utils/init.lua

```diff
@@ -88,11 +88,16 @@ function utils.toggle_autoformat()
 end
 
 function utils.reload_lv_config()
+  -- Hot reload of statusline
```

Comment:
Comment is not accurate, it doesn't hot reload anything, it configures it, and since the code is explicit enough, the comment is not useful

Reply:
Will delete comment